lib/dialogs/buy/bloc/buy_dialog_bloc.dart

```diff
@@ -86,7 +98,16 @@ class BuyDialogBloc extends Bloc<BuyDialogEvent, BuyDialogState> {
       add(OnNewAxInput(axInputAmount: maxInput));
     } catch (e) {
       // TODO(anyone): Create User facing error messages https://athletex.atlassian.net/browse/AX-466
-      emit(state.copyWith(status: BlocStatus.error));
+      emit(
+        state.copyWith(
+          status: BlocStatus.error,
+        ),
+      );
```

Comment:
when dealing with a single prop you could format this a bit better like: 
```
emit(
    state.copyWith(status: BlocStatus.error),
);
```
or even:

```
emit(state.copyWith(status: BlocStatus.error));
```